infrastructure/router/examples/aws-fargate/main.tf

```diff
@@ -0,0 +1,13 @@
+module "cosmo" {
+  source = "../../modules/aws-fargate"
+
+  name             = "cosmo-router"
+  release          = "0.72.0"
```

Comment:
I'd pass there the full name of the docker image + tag. Some users use a custom registry.